images/router/haproxy/conf/haproxy-config.template

```diff
@@ -269,6 +269,12 @@ backend be_edge_http:{{$cfgIdx}}
   balance {{ if gt $cfg.ActiveServiceUnits 1 }}roundrobin{{ else }}leastconn{{ end }}
       {{- end }}
     {{- end }}
+    {{- with $whiteList := index $cfg.Annotations "haproxy.router.openshift.io/whitelist" }}
```

Comment:
I think this name needs to be a little clearer.  Maybe ip_whitelist?  Or allow_ips....